tutorials/hana-cloud-dl-clients-overview/hana-cloud-dl-clients-overview.md

```diff
@@ -38,15 +37,14 @@ SAP HANA Cloud is composed of multiple components.
 
     >Note, that the data lake Files component is currently not available in trial accounts.
 
-For additional details see [How & When to Use the New SAP HANA Cloud Database Instances](https://saphanajourney.com/hana-cloud/resources/sap-hana-cloud-new-services-adaptive-server-enterprise-and-data-lake/).
 
 [DONE]
 [ACCORDION-END]
 
 
 [ACCORDION-BEGIN [Step 2: ](Create a data lake instance and connect to it)]
 
->To complete the tutorials in this group, an SAP HANA Cloud, data lake instance is needed. There are two different free options including SAP HANA Cloud free tier and free trial.  For instructions on registering, see [Set Up Your SAP HANA Cloud, SAP HANA Database (free tier or trial) and Understand the Basics](group.hana-cloud-get-started-1-trial).
+>To complete the tutorials in this group, an SAP HANA Cloud, data lake instance is needed which is created within the SAP Business Technology Platform (BTP). There are two different free options available which are the SAP BTP free-tier and SAP BTP trial.  For instructions on registering, see [Start Using SAP HANA Cloud Free Tier Model or Trial in SAP BTP Cockpit](hana-cloud-mission-trial-1).

```

Comment:
Small typos: 
- It you be "..., a SAP HANA Cloud,..."
- missing comma before "which" (2x)